astropy/wcs/__init__.py

```diff
@@ -3,7 +3,7 @@
 .. _wcslib: https://www.atnf.csiro.au/people/mcalabre/WCS/wcslib/index.html
 .. _distortion paper: https://www.atnf.csiro.au/people/mcalabre/WCS/dcs_20040422.pdf
 .. _SIP: https://irsa.ipac.caltech.edu/data/SPITZER/docs/files/spitzer/shupeADASS.pdf
-.. _FITS WCS standard: https://fits.gsfc.nasa.gov/fits_wcs.html
+.. _FITS WCS standard: https://fits.gsfc.nasa.gov/fits_wcs.html.
```

Comment:
Again a bad one. Maybe make a title sentence as you did elsewhere...

Reply:
✅